client/reader/notifications/controller.jsx

```diff
@@ -11,15 +12,18 @@ export function notifications( context, next ) {
 	recordTrack( 'calypso_reader_notifications_viewed' );
 
 	context.primary = (
-		<div className="reader-notifications__panel">
-			<AsyncLoad
-				require="calypso/notifications"
-				isShowing={ true }
-				checkToggle={ () => {} }
-				setIndicator={ () => {} }
-				placeholder={ null }
-			/>
-		</div>
+		<>
+			<DocumentHead title="Notifications" />
```

Comment:
I'll update that. The other documenthead's I saw in Reader didn't localize either so assumed this was intentional.

Reply:
> The other documenthead's I saw in Reader didn't localize either so assumed this was intentional.

I think this needs to be updated, otherwise the Reader titles won't be properly translated when using a different UI language.